### ISSUE TITLE
How to add DC current source?

### ISSUE THREAD

**Issue by p00f** [drahnr/oregano#248]
Stupid question but how do I add a DC current source? I can only see ISIN

**Comment by drahnr** [comment]
For me there is VDC
![image](https://user-images.githubusercontent.com/667047/104705931-c8872b80-571a-11eb-83eb-1b2590cbf901.png)
with the copr package oregano-0.84.22-1.fc29.x86_64

**Comment by drahnr** [comment]
Feel free to re-open if this does not solve your issue.

**Comment by p00f** [comment]
Yeah VDC is a voltage source right? I'd like to add a current source (that is current in its branch is always the specified value)

**Comment by p00f** [comment]
Yeah VDC is a voltage source right? I'd like to add a current source (that is current in its branch is always the specified value)

**Comment by drahnr** [comment]
Mea culpa.

Oddly only voltage or current _controlled_ current sources exist, so the cleanup of the parts definitions might have cleaned up a few too many.
